dev_ops/infrastructure/dev_resources.yml

```diff
@@ -0,0 +1,17 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template used to create dev resources needed when developing against StreamerContracts
+
+Resources:
+  ExpiredDonationEvent:
+    Type: AWS::Events::Rule
+    Properties:
+      ScheduleExpression: rate(30 minutes)
```

Comment:
So if I understand this correctly, we will poll this queue every 30 minutes? So within this 30 minute window, there's the potential where the contract should be expired, but it hasn't yet because we haven't polled the queue. Is this correct?

Reply:
nawww. Ill try and give you a call to clarify, but this is publishing a message to the queue every 30 minutes. Our application polls this queue (maybe every 15 seconds based on testing) and when a message is found, our application runs whatever logic is annotated with @SqsListener("ExpiredDonationsSQS-${application.environment}")